resolwe/flow/tests/processes/secrets.yml

```diff
@@ -0,0 +1,33 @@
+- slug: test-secrets-permission-denied
+  name: Secret Process
+  version: 1.0.0
+  type: "data:test:secret"
+  input:
+    - name: token
+      label: Secret token
+      type: basic:secret
+  run:
+    program: |
+      # This process should never execute as it lacks the proper permissions
+      # to access any secrets it is passed on input
+
+- slug: test-secrets-echo
+  name: Secret Process
+  version: 1.0.0
+  type: "data:test:secret"
+  requirements:
+    expression-engine: jinja
+    resources:
+      secrets: true
+  input:
+    - name: token
+      label: Secret token
+      type: basic:secret
+  output:
+    - name: secret
+      label: Secret value
+      type: basic:string
+  run:
+    program: |
+      # Echo secret to output. NEVER do this in an actual process!
+      re-save secret "$(cat /secrets/{{token.handle}})"
```

Comment:
It would be useful to add Jinja filter for this, i.e. `{{ token | get_secret }}`

Reply:
For which part? The whole command is specific for bash. Or did you mean just the `{{token.handle}}`?

Reply:
I mean that you write `{{ token | get_secret }}` in the process and it is replaced with `$(cat /secrets/{{token.handle}})` by Jinja filter.

That would make it much more user friendly.